robot/basestation/static/js/elements/feed/camera.js

```diff
@@ -340,4 +340,8 @@ $(document).ready(() => {
       })
     }
   })
+
+  $('.camera-popup' ).click(function() {
+    window.open('/camerapopup',"", 'height='+screen.height+', width='+screen.width);
```

Comment:
Since we are following the standard-js conventions [here](https://github.com/standard/standard/blob/master/RULES.md), infix operators must be spaced.

So for example `var1 + var2 + var3` rather than `var1+var2+var3`

Reply:
Also, after each comma there should be a space too